automation/tox.ini

```diff
@@ -0,0 +1,35 @@
+[tox]
```

Comment:
Please remove this file, I will add one common tox file for all ansible roles

Reply:
done